### DISCUSSION TITLE
Dependabot unlocks transitive dependencies

### DISCUSSION THREAD

**Discussion by SalimBensiali** [community/community#32887] [Code Security] [answered]
I have a question regarding this exciting recent update https://github.blog/changelog/2022-09-07-dependabot-unlocks-transitive-dependencies-for-npm-projects/

Is this supported for both npm and yarn or is it for npm only? If it is the latter, are there any plans to support yarn?

Thanks

**Comment by jhutchings1** [comment, marked as answer]
This feature is only supported for npm at this time. We're working on expanding the set of platforms in the future, and yarn is at the top of that list.

**Reply by SalimBensiali** [reply]
Thank you! Looking forward to this being supported for yarn.
